src/components/Modal/node/OpenChannelModal.tsx

```
@@ -99,6 +108,13 @@
     );
   };
 
+  function handleEnter (event: any) {
+    if (openChannelModal && canOpen && event.key === 'Enter') {
+      console.log('OpenChannelModal event');
+      handleAction();
+    }
+  };
```

ESLint

[error] Unnecessary semicolon.

(no-extra-semi)